app/string-tester.hoon

```diff
@@ -1,12 +1,8 @@
 ::  Primitive tester for the String library.
 /+  string
 !:
-|%
-++  move  {bone card}
-++  card  $%  ~
-          ==
---
-|_  {bowl ~}
+|_  {bowl:gall $~}
+
```

Comment:
While this line *is* in my repo's version, it really shouldn't be there. Whitelines begone!

Reply:
Ok, I'll fix it in both. 😄  I was going to send a small Readme update for that repo as well.